SigningService/src/DataBlock.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace SigningService
```

Comment:
This should not exist in the root project directory. Lets move it into \Signers\StrongName\